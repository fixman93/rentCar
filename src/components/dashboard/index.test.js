import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import { shallow, configure } from 'enzyme'

import { Dashboard } from './index'

configure({ adapter: new Adapter() });

describe('Dashboard', () => {
  const mockRemove = jest.fn()
  const props = { addFeedback: mockRemove, projects: [] }
  const dashboard = shallow(<Dashboard {...props} />)
  console.log(dashboard.debug())
  it('render Dashboard', () => {
    expect(dashboard).toMatchSnapshot()
  })

  describe('addFeedback() on click', () => {
    beforeEach(() => {
      dashboard.find('.review-number').simulate('click')

      it('Call the `addFeedback` callback', () => {
        expect(mockRemove).toHaveBeenCalled()
      })
    })
  })
})