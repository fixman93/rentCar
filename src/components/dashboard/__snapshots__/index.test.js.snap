// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard render Dashboard 1`] = `ShallowWrapper {}`;
